scripts/lib/CIME/XML/entry_id.py

```diff
@@ -306,11 +306,13 @@ def _get_value(self, node, attribute=None, resolved=True, subgroup=None):
             logger.debug("No node")
             return val
 
-        logger.debug("Found node {} with attributes {}".format(node.tag , node.attrib))
+        logger.debug("Found node {} with attributes {}".format(self.name(node) , self.attrib(node)))
         if attribute:
+            vals = self.get_optional_child("values", root=node)
```

Comment:
"value" node may be under "values" node. get_element_text is not a recursive search.